server/konnectors/edf.coffee

```diff
@@ -0,0 +1,1368 @@
+                'xsi:schemaLocation': "http://www.edf.fr/commerce/passerelle/" +
+                "css/visualiserAccordCommercial/service/" +
+                "v2 visualiserAccordCommercial.xsd"
+
+            'msg:enteteEntree':
+                'ent:jeton': data.edfToken
+
+            'msg:corpsEntree':
+                'msg:numeroBp': entries.client[0].clientId
+                'msg:numeroAcc': entries.client[0].numeroAcc
+
+
+    edfRequestPost path, body, (err, result) ->
+        return callback err if err
+        try
+            acoElem = getF result["ns:msgReponse"], "ns:corpsSortie", \
+                "ns:listeAccordCommerciaux", "ns:acordcommercial"
+
+            paymentTerms =
+                vendor: 'EDF'
+                clientId: entries.client[0].clientId
+                docTypeVersion: K.docTypeVersion
+
+            paymentTerms.bankDetails =
+                iban: getF(acoElem, 'ns:banque', 'ns:iban')
+                holder: getF(acoElem, 'ns:compte', 'ns:titulaire')
+                bank: getF(acoElem, "ns:banque", "ns:nom")
+
+            bankAddress =
+                street: getF(acoElem, 'ns:banque', 'ns:numNomRue')
+                city: getF(acoElem, 'ns:banque', 'ns:codePostalVille')
+                # postcode ?
+                country: getF(acoElem, 'ns:banque', 'ns:pays')
+
+            bankAddress.formated = "#{bankAddress.street}" +
+                "\n#{bankAddress.city} #{bankAddress.country}"
+            paymentTerms.bankDetails.bankAddress = bankAddress
+
+            paymentTerms.balance = getF acoElem, 'ns:detail', 'ns:solde'
+            paymentTerms.paymentMeans = getF acoElem, 'ns:detail'
+            , 'ns:modeEncaissement'
+            paymentTerms.modifBankDetailsAllowed = getF acoElem, 'ns:detail'
+            , 'ns:modifIbanAutorisee'
+            #accountNumber: getF acoElem, 'ns:detail', 'ns:numeroEtendu'
+            paymentTerms.dernierReglement =
+                    date: getF acoElem, 'ns:dernierReglement', 'ns:date'
+                    amount: getF acoElem, 'ns:dernierReglement', 'ns:montant'
+                    type: getF acoElem, 'ns:dernierReglement', 'ns:type'
+            paymentTerms.billFrequency = getF acoElem, 'ns:facturation', \
+                                    'ns:periodicite'
+            paymentTerms.nextBillDate = getF acoElem
+                , 'ns:facturation', 'ns:dateProchaineFacture'
+
+            paymentTerms.idPayer = getF acoElem, 'ns:numeroPayeur'
+            paymentTerms.payerDivergent = getF acoElem, 'ns:payeurDivergent'
+
+            servicesElem = acoElem['ns:services']
+            services = servicesElem.map (serviceElem) ->
+                service = {}
+                service.name = getF serviceElem, 'ns:nomService'
+                service.status = getF serviceElem, 'ns:etat'
+                service.valeurSouscrite = getF serviceElem, 'ns:valeurSouscrite'
+                service.valeurPossibles = serviceElem['ns:valeursPossibles']
```

Comment:
Could you translate those properties names ? If not there is a typo I think (a s is missing between valeur and Possibles)